docs/connect-to-lms.md

```diff
@@ -8,6 +8,10 @@ GitHub Classroom integrates with your institution's learning management system (
 
 Classroom works with any LMS that uses the Learning Tools Interoperability (LTI) standard. Using LTI keeps your information safe and secure. LTI is an industry-standard protocol and Classroom's use of LTI is certified by the Instructional Management System Global Learning Consortium.
```

Comment:
Should I remove the `Classroom works with any LMS that uses the Learning Tools Interoperability (LTI) standard`?